.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: ruby
+rvm:
+- 2.2
+
+before_script:
+ - chmod +x ./bin/cibuild # or do this locally and commit
+
+# Assume bundler is being used, therefore
+# # the `install` step will run `bundle install` by default.
+script: ./bin/cibuild
+
+# # branch whitelist, only for GitHub Pages
+branches:
+  only:
+  - gh-pages     # test the gh-pages branch
+  - /pages-(.*)/ # test every branch which starts with "pages-"
```

Comment:
It turns out it will run tests for all PR branches (no matter the name), but for some reason the guidance is to explicitly state `gh-pages`.

[From the docs:](https://docs.travis-ci.com/user/customizing-the-build#Building-Specific-Branches)
>By default, the gh-pages branch is not built unless you add it to the whitelist.